code/modules/research/anomaly/anomaly_core.dm

```diff
@@ -0,0 +1,63 @@
+// Embedded signaller used in anomalies.
+/obj/item/assembly/signaler/anomaly
+	name = "anomaly core"
+	desc = "The neutralized core of an anomaly. It'd probably be valuable for research."
+	icon_state = "anomaly_core"
+	item_state = "electronic"
+	lefthand_file = 'icons/mob/inhands/misc/devices_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
+	resistance_flags = FIRE_PROOF
+	var/anomaly_type = /obj/effect/anomaly
+
+/obj/item/assembly/signaler/anomaly/receive_signal(datum/signal/signal)
+	if(!signal)
+		return FALSE
+	if(signal.data["code"] != code)
+		return FALSE
+	if(suicider)
+		manual_suicide(suicider)
+	for(var/obj/effect/anomaly/A in get_turf(src))
+		A.anomalyNeutralize()
```

Comment:
I don't really like this, storing a weakref to the linked anomaly would probably be better or otherwise use locate()